osu.Game/Database/ImportTask.cs

```diff
@@ -33,7 +33,7 @@ public ImportTask(string path)
         }
 
         /// <summary>
-        /// Construct a new import task from a stream.
+        /// Construct a new import task from a stream. The provided stream will be disposed after reading.
```

Comment:
Bit late here but this is only true if `stream` is not a `MemoryStream`. The new invariant also means that `ImportTask.GetReader()` can only ever be called once successfully. Not a huge deal in current usages, but eventually it may be better to have it implement `IDisposable`.

Reply:
> Bit late here but this is only true if `stream` is not a `MemoryStream`.

`MemoryStream` is a bit of a special case which doesn't actually require disposal. Not sure if it's worth mentioning.

> The new invariant also means that `ImportTask.GetReader()` can only ever be called once successfully.

Fair point, I thought this was already the case because the stream was not being rewound, but turns out it is in `ReadAllBytesToArray`. if it ever becomes an issue we can store the `MemoryStream` to a field I guess.